dolphinscheduler-ui/pom.xml

```diff
@@ -89,61 +89,5 @@
 
       </build>
     </profile>
-
-    <profile>
-      <id>rpmbuild</id>
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>com.github.eirslett</groupId>
-            <artifactId>frontend-maven-plugin</artifactId>
-            <version>${frontend-maven-plugin.version}</version>
-            <executions>
-              <execution>
```

Comment:
Why delete this?

Reply:
This profile it also work for ambari plugin, I can not found any others context link to this.

Reply:
I compare two profile in ui/pom.xml, they are totally the same except profile name.

Reply:
you are right~